staging/src/k8s.io/client-go/tools/leaderelection/metrics.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package leaderelection
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	leaderGauge = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "leader_election_master_gauge",
+			Help: "Gauge of if the reporting system is master of the relevant election.",
```

Comment:
"Count of components that the reporting system is the elected of. Each component leadership is labeled with a name." ?